lib/Firtool/Firtool.cpp

```diff
@@ -142,6 +142,9 @@ LogicalResult firtool::populateCHIRRTLToLowFIRRTL(mlir::PassManager &pm,
 
   pm.nest<firrtl::CircuitOp>().addPass(firrtl::createInlinerPass());
 
+  pm.nest<firrtl::CircuitOp>().addPass(
```

Comment:
You can allow for the pass to run in parallel on all modules by nesting it on FModuleLike rather than CircuitOp, which would improve the performance.